### PR TITLE
Fix Flat upload of assets

### PR DESCRIPTION
**Pull request by karajas**
/cc @wtgodbe @weshaggard

### CODE REVIEW

File: src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeedAction.cs [hunk 1]
```diff
@@ -90,13 +90,16 @@ public BlobFeedAction(string expectedFeedUrl, string accountKey, string indexDir
 
                 await UploadAsync(CancellationToken, item, uploadPath, clientThrottle, allowOverwrite);
 
-                List<string> listAzureBlobs = await ListAzureBlobs.ListBlobs(Log, feed.AccountName, feed.AccountKey, feed.ContainerName, packageDirectory);
-                if (!listAzureBlobs.Any(x => x.Contains(uploadPath)))
+                if (!isFlat)
```
Comment (karajas) [inline src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeedAction.cs]: Skipping generation of index.json uploads

File: src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeed.cs [hunk 2]
```diff
@@ -153,13 +153,13 @@ public string CalculateBlobPath(string item, string relativePath)
...
                 Tuple<string, NuGetVersion> nugetPackage = GetNugetPackageVersion(item);
-                blobPath = Path.Combine(Path.Combine(relativePath, Path.Combine(nugetPackage.Item1, nugetPackage.Item2.ToFullString())), Path.GetFileName(item));
+                blobPath = Path.Combine(Path.Combine(relativePath, Path.Combine(nugetPackage.Item1, nugetPackage.Item2.ToFullString())), Path.GetFileName(item)).ToLowerInvariant();
             }
             else
             {
-                blobPath = item.Substring(item.IndexOf(relativePath));
+                blobPath = Path.Combine(relativePath, Path.GetFileName(item));
             }
-            return blobPath.ToLowerInvariant().Replace("\\", "/");
```
Comment (karajas) [inline src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeed.cs]: Moving toLower to just the nupkg section to preserve cases for files

**Comment by karajas** [comment]
This also assumes that we can only upload flat files, never folders.

File: src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeed.cs (hunk 2 above)
Comment (weshaggard) [reply src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeed.cs]: why does the path need to be tolower for the nuget files?

**Pull request merged**